README.md

```diff
@@ -2,145 +2,132 @@
 
 [![Build Status](https://travis-ci.com/makerdao/vdb-mcd-transformers.svg?token=yVLfQ4hNQCqvt2qjLkus&branch=staging)](https://travis-ci.com/makerdao/vdb-mcd-transformers)
 
-## About
 
-Vulcanize DB is a set of tools that make it easier for developers to write application-specific indexes and caches for dapps built on Ethereum.
-
-## Dependencies
- - Go 1.11+
- - Postgres 11
+## Table of Contents
+1. [Background](#background)
+1. [Install](#install)
+1. [Usage](#usage)
+1. [Contributing](#contributing)
+1. [License](#license)
+
+## Background
+This repository is a collection of transformers to be used along with VDB as a plugin to fetch, transform and persist log events and storage slots of
+specified MCD contracts.
+
+## Install
+### Dependencies
+ - [VulcanizeDB](https://github.com/makerdao/vulcanizedb)
+ - Go 1.12+
+ - Postgres 11.2
  - Ethereum Node
-   - [Go Ethereum](https://ethereum.github.io/go-ethereum/downloads/) (1.8.23)
+   - [Go Ethereum](https://ethereum.github.io/go-ethereum/downloads/) (1.8.23+)
    - [Parity 1.8.11+](https://github.com/paritytech/parity/releases)
-
-## Project Setup
-
-Using Vulcanize for the first time requires several steps be done in order to allow use of the software. The following instructions will offer a guide through the steps of the process:
-
-1. Fetching the project
-2. Installing dependencies
-3. Configuring shell environment
-4. Database setup
-5. Configuring synced Ethereum node integration
-6. Data syncing
-
-## Installation
-
-In order to fetch the project codebase for local use or modification, install it to your `GOPATH` via:
-
-`go get github.com/makerdao/vulcanizedb`
-`go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace`
-
-Once fetched, dependencies can be installed via `go get` or (the preferred method) at specific versions via `golang/dep`, the prototype golang pakcage manager. Installation instructions are [here](https://golang.github.io/dep/docs/installation.html).
-
-In order to install packages with `dep`, ensure you are in the project directory now within your `GOPATH` (default location is `~/go/src/github.com/makerdao/vulcanizedb/`) and run:
-
-`dep ensure`
-
-After `dep` finishes, dependencies should be installed within your `GOPATH` at the versions specified in `Gopkg.toml`.
-
-Lastly, ensure that `GOPATH` is defined in your shell. If necessary, `GOPATH` can be set in `~/.bashrc` or `~/.bash_profile`, depending upon your system. It can be additionally helpful to add `$GOPATH/bin` to your shell's `$PATH`.
-
-## Setting up the Database
+   
+### Getting the project
+Download the transformer codebase to your local local `GOPATH` via:
+`go get github.com/makerdao/vdb-mcd-transformers`
```

Comment:
Does this work? I thought we might not be able to use `go get` as long as we're importing the Geth fork

Reply:
Good call, I had forgotten about that. I think `go get` should work for getting the transformers repo - since it doesn't directly depend on geth, but geth through VDB. But you're totally right that we'll probably have to use `git clone` for the vdb repo.

That said, I wonder if we should use the same method for both repos, just for clarity/consistency?